projects/admin/src/app/record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component.html

```diff
@@ -177,56 +188,102 @@ <h1 translate>Circulation Policy</h1>
 
     <!-- start: notifications and fees section !-->
     <fieldset class="mt-4" *ngIf="allow_checkout.value">
-      <div class="row">
-        <legend translate>Notifications and fees</legend>
-        <div class="col-sm-12 pt-4">
-          <div class="form-group row" *ngIf="allow_checkout.value">
-            <label for="daysAfterControl" class="col-sm-2 col-form-label required" translate>First reminder days</label>
+      <div class="card">
+        <div class="card-header" translate>Notifications and fees</div>
+        <div class="card-body">
+          <!-- Sent loan expiry notice-->
+          <div class="form-group row">
             <div class="col-sm-4">
-              <input type="number" class="form-control" id="daysAfterControl"
-                formControlName="number_of_days_after_due_date">
-              <div
-                *ngIf="number_of_days_after_due_date.invalid && (number_of_days_after_due_date.dirty || number_of_days_after_due_date.touched)"
-                class="text-danger pt-1">
-                <div *ngIf="number_of_days_after_due_date.errors.required" translate>
-                  First reminder days is required.
+              <div class='form-group'>
+                <div class="custom-control custom-switch">
+                  <input
+                    formControlName="loan_expiry_notice"
+                    class="custom-control-input"
+                    type="checkbox"
+                    id="loan_expiry_notice"
+                    [checked]="loan_expiry_notice.value">
+                  <label class="custom-control-label" for="loan_expiry_notice" translate>
+                    Send a loan expiry notice
+                  </label>
                 </div>
-                <div *ngIf="number_of_days_after_due_date.errors.min" translate>
-                  First reminder days must be greater than 0.
+              </div>
+            </div>
+            <div class="col-sm-8 border rounded pt-3" *ngIf="loan_expiry_notice.value">
+              <div class="form-group row">
+                <label class="col-sm-6 col-form-label text-right" for="daysBeforeControl" translate>
+                  Sending how many days before the due date
+                </label>
+                <input type="number" class="form-control col-sm-3" id="daysBeforeControl"
+                  formControlName="number_of_days_before_due_date">
+                <div
+                  *ngIf="number_of_days_before_due_date.invalid && (number_of_days_before_due_date.dirty || number_of_days_before_due_date.touched)"
+                  class="text-danger pt-1">
+                  <div *ngIf="number_of_days_before_due_date.errors.required" translate>
+                    Days before due date is required.
```

Comment:
```suggestion
                    The number of days before due date a notification is sent is required.
```
I think this should be checked with @pronguen